src/components/table/table.tsx

```diff
@@ -182,14 +194,25 @@ export class Table {
     }
 
     @Watch('columns')
-    public updateColumns() {
+    public updateColumns(newColumns: Column[], oldColumns: Column[]) {
         if (!this.tabulator) {
             return;
         }
 
+        if (this.isReordered(newColumns, oldColumns)) {
+            return;
+        }
+
         this.tabulator.setColumns(this.getColumnDefinitions());
     }
 
+    private isReordered(newColumns: Column[], oldColumns: Column[]) {
```

Comment:
I am not sure if the function name is good name for what it does. This function returns a boolean whether all elements in the `newColumns`  exist in `oldColumns` . This doesn't show if there is a reorder or not.

This is how I check if there is reorder 

```
for (var i = 0; newColumns.length < i; i++) {
		if (newColumns[i] !== oldColumns[i]) return true;
	}
```
	
	My point is shortly, we probably give a better name for the function.

Reply:
I changed the name of the method